Sources/Sentry/SentryTracer.m

```diff
@@ -87,6 +87,8 @@ @implementation SentryTracer {
 static NSObject *appStartMeasurementLock;
 static BOOL appStartMeasurementRead;
 
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wshadow"
```

Comment:
`m`: Why do we have to add this while there are no changes in between?
```
#pragma clang diagnostic push
#pragma clang diagnostic ignored "-Wshadow"
```

and
```
#pragma clang diagnostic pop
```

Reply:
Probably some test left behind.